src/styles/main.scss

```
@@ -1,3 +1,34 @@
-body {
+@import "./header/style.scss";
+@import "./card/index.scss";
+
+html {
   margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__catalog {
+    display: grid;
+    grid-template-columns: repeat(4, 200px);
+    grid-template-rows: repeat(2, 408px);
+    gap: 46px 48px;
+    padding: 50px 40px;
+  }
+
+  @media (max-width: 768px) {
+    &__catalog {
+      display: grid;
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
```

```suggestion
  &__catalog {
    display: grid;
    grid-template-columns: repeat(4, 200px);
    gap: 46px 48px;
    padding: 50px 40px;
    
    @media (max-width: 768px) {
      grid-template-columns: repeat(3, 200px);
    }
  }
```
- you already specified `display: grid;` to `&__catalog`.
- you don't need to repeat selectors in scss
- you don't need to specify `grid-template-rows: repeat(2, 408px);` as it could be more than 2 rows